core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -32,6 +32,14 @@
 #define KOKKOS_IMPL_OPENMPTARGET_HIERARCHICAL_INTEL_GPU
 #endif
 
+// Define a macro for llvm compiler greater than version 15 and on NVIDIA and
+// AMD GPUs. This would be useful in cases where non-OpenMP standard llvm
+// extensions can be used.
+#if defined(KOKKOS_COMPILER_CLANG) && (KOKKOS_COMPILER_CLANG >= 1500) && \
+    (defined(KOKKOS_ARCH_AMD_GPU) || defined(KOKKOS_IMPL_ARCH_NVIDIA_GPU))
+#define KOKKOS_IMPL_OPENMPTARGET_LLVM_EXTENSIONS
```

Comment:
Is that something you would want to see in print_configuration?
If we were getting a bug report, would you care to know about that setting?

Reply:
Thats a good idea. I will include it there.

Reply:
Ping